cables.py

```diff
@@ -64,35 +64,31 @@ def get_cables_on_pages(list_of_page_elements):
         if not cables_list:
             logging.info(f'There are no cables on page {page_name}')
             continue
-        logging.info(f'page_name: {page_name}')
-        for cable_dict in cables_list: 
-            logging.info(f'(cable_dict):{cable_dict}')
+        logging.info(f'found {len(cables_list)} cables on page {page_name}')
         number_of_cables += len(cables_list)
-        logging.info(f'amount of cables: {len(cables_list)}')
         if json_output:
             create_json(cables_list,page_name)
     return list_of_cable_elements, cables_list
 
 def set_cables_on_pages(list_of_page_elements, cables_list, last_number):
     number_of_cables = 0
     new_number = last_number + 1
+
     for page_elements in list_of_page_elements:
         page_name = page_elements.get('name')
         list_of_cable_elements = get_cable_elements(page_elements)
         cables_list = get_cable_list(page_elements, list_of_cable_elements, page_name)
         if not cables_list:
             logging.info(f'There are no cables on page {page_name}')
             continue
+
         new_number = set_new_cable_label(page_elements, cables_list, new_number)
-        cables_list = get_cable_list(page_elements, list_of_cable_elements, page_name)
```

Comment:
If we don't check again for the updated cable numbers we can't write a cable list with the same...

Reply:
good that you found it, was not sure about the need for calling it twice!